docs/cli/login-success.txt

```diff
@@ -0,0 +1,12 @@
+
+{bold Deploying a Git Repository with ZEIT Now}
```

Comment:
```suggestion
{bold ProTip!} Deploy a Git Repository
```

Now that I think about it, this feature is very similar to the "ProTip" from GitHub. Scroll to the bottom of any github.com page like this PR and you'll see a randomly generated ProTip.